roles/remediate/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# defaults file for remedations
```

Comment:
Need to add `post_reboot_delay: 120` here.

Reply:
Done.